src/include/security/widgets.rb

```diff
@@ -261,6 +247,22 @@ def initialize_security_widgets(include_target)
             "Value"  => "100"
           }
         }
+
+      @WIDGETS.merge!(
+        @display_manager.shutdown_var_name => shutdown_login_manager_widget
+      ) if @display_manager
+    end
+
+    def shutdown_login_manager_widget
+      {
+        "Widget"  => "ComboBox",
+        # ComboBox label
+        "Label"   => _(
+          "&Shutdown Behaviour of %s Login Manager:"
```

Comment:
OK, i will add, it was GDM and KDM before so it should be already present, but it could be other LM.